Assets/_Laz/Scripts/Laz/LazCoordinatorBehaviour.cs

```diff
@@ -75,5 +78,11 @@ public void Reset()
             _boostBehaviour.Reset();
             transform.position = _lazPlayer.SpawnPosition;
         }
+
+        public void KillLaz()
+        {
+            //Todo: Handle Death animations and such
+            OnLazDieing?.Invoke();
```

Comment:
NIT: 
OnLazDieing?.Invoke(); 

So this thing i read on Resharper, that because it's inside Monobehaviour, there's a chance that the C# object dies, but the Monobehaviour is still alive and Invokes this.

But i also watch video saying that it doesn't really matter. 


Just a thought.

Reply:
Ooh interesting. No harm to change it then I guess haha